doc/nrf/releases/release-notes-latest.rst

```diff
@@ -331,8 +325,7 @@ The following list summarizes the most important changes inherited from upstream
   * File systems:
 
     * Enabled FCB to work with non-0xff erase value flash.
-    * Enabled FCB to work with non-0xff erase value flash.
-    * Added ``CONFIG_FS_MOUNT_FLAG_NO_FORMAT`` flag to FATFS Kconfig options, that removes formatting capabilities from FAT FS;
-      the option also prevents unformatted devices to be formatted to FAT FS on mount attempt.
-    * Added support for ``fs_mount()`` flags: ``FS_MOUNT_FLAG_READ_ONLY`` and ``FS_MOUNT_FLAG_NO_FORMAT``.
-    * The runtime check of FS API of a driver interface will not be perfromed when ``CONFIG_NO_RUNTIME_CHECKS`` option is enabled.
+    * Added a :c:macro:`FS_MOUNT_FLAG_NO_FORMAT` flag to the FatFs options.
+      This flag removes formatting capabilities from FatFs and prevents unformatted devices to be formatted to FatFs on mount attempt.
```

Comment:
The FatFs here refers FAT file system, where FAT means File Allocation Table this is not name of a library.

Reply:
Maybe, instead of `FatFs`, use `FAT file system`.

Reply:
I'm not sure I understand where you refer to the library and where to the file system - please make a suggestion for the full entry.

Reply:
`This flag removes formatting capabilities from the FAT/exFAT file system driver and prevents unformatted devices to be formatted, to FAT or exFAT, on mount attempt.`

This is FAT https://en.wikipedia.org/wiki/File_Allocation_Table.
This is the library, FatFs, that supports it http://elm-chan.org/fsw/ff/00index_e.html.

The flags are generally for the FAT file system even though we use only one library for it, which is FatFs.
I see this way: we may use Groke named library, which would be supporting the FAT file system, but the user will still be formatting devices to FAT, or mounting FAT formatted devices; the user will not be trying to mount SD card formatted to Groke (that would be scary and would freeze the device anyway).

The devices available on the market, like SD cards, are usually formatted to FAT or exFAT.

Reply:
Subjesct of the first mention of `FatFS` is the library which implements FAT file system, the second mention mean the volume of FAT file system. 
Second mention should reflect that: `FAT file system' is more suitable there.

Reply:
OK, sounds good, please change as suggested. :)